src/l3-resources/atlas-basic-private-endpoint/index.ts

```diff
@@ -84,7 +118,7 @@ export interface AtlasBasicPrivateEndpointProps {
    * @default us-east-1
    * @memberof AtlasPrivateEndpointProps
    */
-  readonly region?: string;
+  readonly region: string;
```

Comment:
why is this prop now required?

Reply:
it should be a mandatory field, it was throwing me an error when i did not add it